### DISCUSSION TITLE
Some extra info/sensors that can be nice

### DISCUSSION THREAD

**Discussion by jeremycoppey** [lbbrhzn/ocpp#48] [Ideas]
Would be nice to have some more info/sensors

- Charge time of session
- The energy usage for the current charging session
- Set charging limit

**Comment by jeremycoppey** [comment]
I think I can already answer 2 of my request.
there are 2 sensors available: 
- session.energy
- session.time
That cover the needs (they don't work on my charger apparently

**Reply by lbbrhzn** [reply]
Right. For the charging limit, there is a proposal from @drc38 in a comment of #34. I'll create a new issue for it though!

**Reply by jeremycoppey** [reply]
Can we close this discussion as the issue https://github.com/lbbrhzn/ocpp/issues/51  has been closed?

**Reply by lbbrhzn** [reply]
Yes, although I don’t think you can close a discussion. It is possible to lock it, or delete the thread, though.
